editor/src/components/canvas/controls/select-mode/flex-reparent-target-indicator.tsx

```diff
@@ -1,26 +1,47 @@
 import React from 'react'
+import { useColorTheme } from '../../../../uuiui'
+import { Utils } from '../../../../uuiui-deps'
 import { useEditorState } from '../../../editor/store/store-hook'
 import { CanvasOffsetWrapper } from '../canvas-offset-wrapper'
 
 export const FlexReparentTargetIndicator = React.memo(() => {
+  const colorTheme = useColorTheme()
+  const [mouseOverSizeOffset, setMouseOverSizeOffset] = React.useState(0)
   const reparentTargetLines = useEditorState(
     (store) => store.editor.canvas.controls.flexReparentTargetLines,
     'FlexReparentTargetIndicator lines',
   )
+
+  const isHoverInteraction = useEditorState(
+    (store) => store.editor.canvas.interactionSession?.interactionData.type === 'HOVER',
+    'FlexReparentTargetIndicator isHoverInteraction',
+  )
+  const mouseover = React.useCallback(() => {
+    setMouseOverSizeOffset(4)
+  }, [setMouseOverSizeOffset])
+  const mouseLeave = React.useCallback(() => {
+    setMouseOverSizeOffset(0)
+  }, [setMouseOverSizeOffset])
+
   return (
     <CanvasOffsetWrapper>
       <div style={{ display: 'block' }}>
         {reparentTargetLines.map((line, i) => (
           <div
+            onMouseOver={isHoverInteraction ? mouseover : Utils.NO_OP}
```

Comment:
I'm not sure if it makes any difference, but it feels like the check against `isHoverInteraction` should be inside `mouseover` and `mouseLeave`.

Reply:
reverted this functionality